src/main.rs

```diff
@@ -733,7 +734,7 @@ mod test_song_parsing {
 
 	#[test]
 	#[should_panic(
-		expected = "called `Result::unwrap()` on an `Err` value: MultipleSegmentsWithSameId(\"format\", \"loop\")"
+		expected = "called `Result::unwrap()` on an `Err` value: MultipleSegmentsWithSameId(\"song_format\", \"loop\")"
```

Comment:
We might want to remove `#[should_panic]` and replace the `initialize_songs(&paths).unwrap();` with something like:
```rust
let result = initialize_songs(&paths);
assert!(!result.is_err());
```

Reply:
It's not really related to this PR though.